src/docs/_includes/navigation.html

```diff
@@ -1,9 +1,12 @@
 <nav class="main-menu">
+    <a href="./" class="main-menu__home">
+        <img src="images/logos/brainly-mobile.svg" />
+    </a>
```

Comment:
Logo at the beginning of the navigation is now clickable and points to the intro page.